docs/docs/contribute/code/building-shards.md

```diff
@@ -195,7 +226,11 @@ You need to run the following two commands every time you want to build the proj
 Configure the build with `cmake`,
 
 ```
+# Windows
 cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug -DCMAKE_EXPORT_COMPILE_COMMANDS=1 ..
```

Comment:
`CMAKE_EXPORT_COMPILE_COMMANDS=1` is not necessary in either.

Reply:
yeah, I was wondering about that...let me remove it.